ya-runtime-erigon/Cargo.toml

```diff
@@ -24,4 +24,4 @@ titlecase = "1.1.0"
 tokio = { version = "0.2", features = ["io-std", "io-util", "rt-core", "rt-threaded", "macros", "process", "time"] }
 
 [patch.crates-io]
-ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "880efaf19111e71647dea23fd997ebf6edc6e3d7" }
+ya-runtime-sdk = "0.1.0"
```

Comment:
Patching is no longer necessary

Reply:
Thanks! Resolved in [[#pull/23/commits/8816394](https://github.com/golemfactory/yagna-service-erigon/pull/23/commits/8816394ccbd6a103ed41a9afc59e4773c4750eb6)]